exercises/complex-numbers/src/example/kotlin/ComplexNumber.kt

```diff
@@ -0,0 +1,26 @@
+data class ComplexNumber(val real: Double = 0.0, val imag: Double = 0.0)
+
+operator fun ComplexNumber.plus(other: ComplexNumber)
```

Comment:
Would it make sense to put these functions inside the class instead of making them extension functions?